### PR TITLE
Version 0.23.0

### PR DESCRIPTION
**Pull request by cbracken**

### CODE REVIEW

File: lib/strings.dart [hunk 1]
```diff
@@ -46,11 +46,13 @@ String flip(String s) {
 /**
  * If [s] is [null] returns empty string, otherwise returns [s] as is.
  */
+@Deprecated('Will be removed in 0.23.0')
```
Comment (yjbanov) [inline lib/strings.dart]: Let's suggest an alternative in the deprecation message.

File: lib/strings.dart [hunk 2]
```diff
@@ -46,11 +46,13 @@ String flip(String s) {
...
 String nullToEmpty(String s) => s == null ? '' : s;
 
 /**
  * If [s] is an empty string returns [null], otherwise returns [s] as is.
  */
+@Deprecated('Will be removed in 0.23.0')
```
Comment (yjbanov) [inline lib/strings.dart]: ditto

File: lib/strings.dart (hunk 1 above)
Comment (cbracken) [inline lib/strings.dart]: Done.

File: lib/strings.dart (hunk 2 above)
Comment (cbracken) [inline lib/strings.dart]: Done.

**Comment by xster** [comment]
The alternative for emptyToNull is still quite verbose no?
Consider the string `someList.where(...).map(...).join(...)`. It's hard to code golf it to one line of non-empty otherwise.

**Pull request merged**